common/api/components-react.api.md

```diff
@@ -260,6 +260,7 @@ export interface CommonPropertyGridProps extends CommonProps {
     onPropertyUpdated?: (args: PropertyUpdatedArgs, category: PropertyCategory) => Promise<boolean>;
     orientation?: Orientation;
     propertyValueRendererManager?: PropertyValueRendererManager;
+    showOnlyEditor?: (property: PropertyRecord) => boolean;
```

Comment:
What do you think about naming this: `alwaysShowEditor`? `showOnlyEditor` sounds a little bit misleading as property label is also shown. 

Also can you add a story with some editors always visible? At least for default checkbox and toggle editors.

Reply:
added a story, which revealed a bug, added test to cover it.